src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java

```diff
@@ -262,6 +262,120 @@ public void testSafeRunnable()
+    public void testSafeRunnableInMessage()
+    {
+        // preparing the test setup as kind of generator
+        List<Runnable> runArray = new ArrayList<Runnable>();
+        runArray.add(
+                     () -> {
+                         throw UIAssertionError.wrap(WebDriverRunner.driver(),
+                                                     new AssertionError(" value displayed:StaleElementReferenceException: stale element reference: element is not attached to the page document>StaleElementReferenceException: stale element reference: element is not attached to the page document</StaleElementReferenceException: stale element reference: element is not attached to the page document>'\n"
+                                                                        +
+                                                                        "Actual value: StaleElementReferenceException: stale element refe"),
+                                                     0);
+                     });
+        runArray.add(
+                     () -> {
+                         throw UIAssertionError.wrap(WebDriverRunner.driver(),
+                                                     new AssertionError(" value displayed:StaleElementReferenceException: stale element reference: element is not attached to the page document>StaleElementReferenceException: stale element reference: element is not attached to the page document</StaleElementReferenceException: stale element reference: element is not attached to the page document>'\n"
+                                                                        +
+                                                                        "Actual value: StaleElementReferenceException: stale element refe"),
+                                                     0);
+                     });
+        runArray.add(
+                     () -> {
+                         throw UIAssertionError.wrap(WebDriverRunner.driver(),
+                                                     new AssertionError(" value displayed:StaleElementReferenceException: stale element reference: element is not attached to the page document>StaleElementReferenceException: stale element reference: element is not attached to the page document</StaleElementReferenceException: stale element reference: element is not attached to the page document>'\n"
+                                                                        +
+                                                                        "Actual value: StaleElementReferenceException: stale element refe"),
+                                                     0);
+                     });
+        runArray.add(
+                     () -> {
+                         throw new StaleElementReferenceException("You shall pass!");
+                     });
+        runArray.add(
+                     () -> {
+                         throw new StaleElementReferenceException("You shall never be seen!");
```

Comment:
Please change this also to be an UIAssertionError

Reply:
Done. The text "StaleElementException" added to avoid failues on the further assertions for supplier (line 384-386)